2022q3/cloud-init.adoc

```diff
@@ -0,0 +1,22 @@
+=== FreeBSD as a Tier I cloud-init Platform
+
+Links: +
+link:https://https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[https://cloud-init.io/] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[https://cloudinit.readthedocs.io/en/latest/] +
+link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[cloud-init ongoing refactorization] URL: link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst] +
+
+Contact: Mina Galić <me+FreeBSD@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Unfortunately, cloud-init’s support for operating systems other than Linux is rather poor and the lack of cloud-init support on FreeBSD is a hindrance to cloud providers who want to offer FreeBSD as a Tier I platform.
+To remedy this, this project aims to bring FreeBSD cloud-init support on par with Linux support. Though the plan is to lift support across all BSDs.
+
+The project deliverables include completing an extraction of certain networking classes, implementing ifconfig(8) and login.conf(5) parsers, implementing IPv6 configuration, creating devd rules for Azure, and writing Handbook documentation about productionizing FreeBSD.
+
+On the way there, any BSD related bugs found in modules and documentation will also be fixed.
+
+People interested in helping with the project can help with testing new features and fixes through the net/cloud-init-devel port, which will be updated on a weekly basis.
+Further, people with access and experience with OpenBSD and NetBSD are also highly welcome to help.
+
+
```

Comment:
Spurious linebreak?

Reply:
i usually break on sentences in such documents, as it makes for easier diffs

Reply:
Sorry, I meant the double one at the end.